modules/product/src/main/java/com/opengamma/strata/product/bond/YieldConvention.java

```diff
@@ -21,7 +21,8 @@
  * specific to each yield convention. Accordingly, the computation of price, convexity
  * and duration from the yield should be based on this yield convention. 
  * <p>
- * Reference: "Bond Pricing", OpenGamma Documentation 5, Version 2.0, May 2013.
+ * References: "Bond Pricing", OpenGamma Documentation 5, Version 2.0, May 2013, 
+ * "Inflation Instruments: Swap Zero-coupon, Year-on-year and Bonds."
  */
 public enum YieldConvention {
```

Comment:
When I looked at OG-Platform Biill yield conventions had nothing in common with bond ones, so I think separate types makes sense.

More broadly, I don't like the `if(convention = FOO)` style, as really the logic should be in the convention, but that is a bigger change.